client/src/components/Button/HelpBoardButton.js

```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
+import BaseButton from "./BaseButton";
```

Comment:
There is a lot of code repetition here, with `HelpBoardButton`, `LinkButton`,  `popUpButton`, please. create one button only and then extend from that one if you need to add more things to another button. + same comments apply for all Button Components created.

Reply:
removed repeated code as much.